include/lemlib/movements/purepursuit.hpp

```diff
@@ -15,15 +15,15 @@ class PurePursuit : public Movement {
         /**
          * @brief Construct a new Pure Pursuit movement
          *
-         * @param trackWidth the width of the chassis. Units in inches
+         * @param trackWidth the width of the chassis.
          * @param path const reference to the asset containing the path
-         * @param lookaheadDist the lookahead distance. Units in inches. Recommended value is 15, but can be changed if
-         * needed
+         * @param lookaheadDist the lookahead distance. Recommended value is 15, but can be changed if needed
```

Comment:
```suggestion
         * @param lookaheadDist the lookahead distance. Recommended value is 15_in, but can be changed if needed
```
Specify units for recommended value